modules/monitoring-adding-a-secret-to-the-alertmanager-configuration.adoc

```diff
@@ -138,3 +138,50 @@ endif::openshift-dedicated,openshift-rosa[]
 
 . Save the file to apply the changes to the `ConfigMap` object. The new configuration is applied automatically.
 
+.Examples
+
+These examples show settings that configure Alertmanager in the `openshift-monitoring` namespace to authenticate with a webhook endpoint by using an API token that is configured in a `Secret` object.
+
+The following sample YAML shows a `Secret` object with authentication credentials to be referenced in the Alertmanager config map settings for the Cluster Monitoring Operator (CMO):
+
+[source,yaml]
+----
+apiVersion: v1
+data: 
+  api_token: m8VjfmV2Cg== <1>
+kind: Secret
+metadata: 
+  name: alertmanager-credentials <2>
+  namespace: openshift-monitoring
+----
+<1> Replace this value with the API token used to authenticate to your webhook endpoint.
+<2> Reference this name in the `secrets` field in the CMO config map settings. 
+
+The following sample YAML shows CMO config map settings for Alertmanager.
+These settings reference the authentication credentials contained in the `Secret` object, which is mounted as a volume in the `alertmanager` container for the Alertmanager pods:
```

Comment:
just wondering if this should say "the previous `Secret` object" to be clear which one we're talking about?

Reply:
Yes. Good point.